webapp/channels/src/components/admin_console/license_settings/modals/ee_license_modal.tsx

```diff
@@ -37,18 +38,20 @@ const EELicenseModal: React.FC<Props> = (props: Props): JSX.Element | null => {
     // Note: DO NOT LOCALISE THESE STRINGS. Legally we can not since the license is in English.
     return (
         <GenericModal
+            modalHeaderText={
+                <FormattedMessage
+                    id='EE_license_modal.defaultMsg'
+                    defaultMessage='Enterprise Edition License'
```

Comment:
Per the comment above, this string must not be localized. Can you please revert the change? Thanks!

Reply:
Sure. I wasn't aware `FormattedMessage` was used to localize strings at the time.